package.json

```diff
@@ -6,6 +6,9 @@
   "author": "Roy Sutton <roy.sutton@lge.com>",
   "repository": "https://github.com/enactjs/jsdoc-to-ts",
   "license": "Apache-2.0",
+  "engines": {
```

Comment:
Is there a reason to specify this node version?
It is different from node-glob, and there is no information on which module you referred to.

Reply:
As per changelog.md in glob, glob is supported from node 16 and above, so 16 to 16.12 is also supported.
https://github.com/isaacs/node-glob/blob/8c371a3eaacaa0783c34bda13d32fca3219017d3/package.json#L96
The '>=16.13.0' specified in the current package.json means discarding 16 to 16.12.
I was wondering why you excluded this 16~16.12 version.
(Is the node version of the other 3rd party module '>=16.13.0'?)
If not, please set the support node version as same as glob.

Reply:
Thank you for this notice, I changed the node version to support "node": ">=16"